src/momento/internal/_utilities/_data_validation.py

```diff
@@ -47,12 +54,26 @@ def _as_bytes(
 
 def _list_as_bytes(
     values: TListValuesInput, error_message: Optional[str] = DEFAULT_LIST_CONVERSION_ERROR
-) -> TListValuesInputBytes:
+) -> Iterable[bytes]:
     if not isinstance(values, collections.abc.Iterable):
         raise InvalidArgumentException(f"{error_message}{type(values)}")
     return [_as_bytes(value) for value in values]
 
 
+def _dictionary_items_as_bytes(
+    items: TDictionaryItems, error_message: Optional[str] = DEFAULT_DICTIONARY_CONVERSION_ERROR
+) -> Dict[bytes, bytes]:
+    if not isinstance(items, collections.abc.Mapping):
+        raise InvalidArgumentException(f"{error_message}{type(items)}")
+    return {_as_bytes(key): _as_bytes(value) for key, value in items.items()}
+
+
+def _dictionary_fields_as_bytes(
+    fields: TDictionaryFields, error_message: Optional[str] = DEFAULT_DICTIONARY_FIELDS_CONVERSION_ERROR
+) -> Iterable[bytes]:
+    return _list_as_bytes(fields, error_message)
+
+
 def _set_as_bytes(
     elements: TSetElementsInput, error_message: Optional[str] = DEFAULT_SET_CONVERSION_ERROR
 ) -> TSetElementsInputBytes:
```

Comment:
If _dictionary_fields_as_bytes and _list_as_bytes are returning `Iterable[bytes]` so should this.

Reply:
Take a look at [49b7bdb](https://github.com/momentohq/client-sdk-python/pull/237/commits/49b7bdbd9338c32504ec513aff3f6706c0ef06bf) for changes to consistently use generators.